build-system/runner/README.md

```diff
@@ -1,4 +1,6 @@
-You need to recompile `dist/runner.jar` for any source code changes made to this dir.
+You need to recompile `dist/runner.jar` on a Linux machine for any source code
```

Comment:
This is not true. It can be compiled on Mac OS too. I routinely compile `runner.jar` on a macbook using:
```sh
rsimha@rsimha-macbookpro2 (master):~/src/amphtml$ ant -version
Apache Ant(TM) version 1.10.5 compiled on July 10 2018
```

Reply:
hmm.. my dist/runner.jar compiled on my Mac OS keeps failing on Linux. Might be different java version? javac 12.0.1

Reply:
Yeah, most likely.
```
rsimha@rsimha-macbookpro2 (master):~/src/amphtml$ javac -version
javac 1.8.0_172
```

Reply:
Curious: Did you follow the [instructions in the dev docs](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#building-amp-and-starting-a-local-server) to choose the correct version of Java?

> Closure Compiler is automatically installed by Yarn, but it requires Java 8 which you need to install separately. AMP's version of Closure Compiler won't run with newer versions of Java. Download an installer for Mac, Linux or Windows here.
> 
> Note: If you are using Mac OS and have multiple versions of Java installed, make sure you are using Java 8 by adding this to `~/.bashrc`:
```
export JAVA_HOME=`/usr/libexec/java_home -v 1.8`
```

Reply:
Missed that. Thanks